packages/frontend/core/src/hooks/affine/use-register-blocksuite-editor-commands.tsx

```diff
@@ -18,10 +18,10 @@ import { useBlockSuiteMetaHelper } from './use-block-suite-meta-helper';
 import { useExportPage } from './use-export-page';
 import { useTrashModalHelper } from './use-trash-modal-helper';
 
-export function useRegisterBlocksuiteEditorCommands(
-  pageId: string,
-  mode: 'page' | 'edgeless'
-) {
+export function useRegisterBlocksuiteEditorCommands() {
+  const page = useService(Page);
```

Comment:
is `useService` reactive? if not, the hooks below use `pageId` as dependency would error

Reply:
useService is reactive